gengo/service_routing.go

```diff
@@ -189,6 +203,12 @@ func generateOperationMethod(w *sources.Writer, operation *spec.NamedOperation)
 
 	generateServiceCall(w, operation)
 
+	if operation.Body != nil && operation.Body.Type.Definition.Plain == spec.TypeString {
+		w.Line(`res.Header().Set("Content-Type", "text/plain")`)
+	} else {
```

Comment:
what if `body` is just `nil` is it good to set `Content-Type` to `application/json` then?